rules/msk_module_backend.go

```diff
@@ -0,0 +1,148 @@
+package rules
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	"github.com/terraform-linters/tflint-plugin-sdk/hclext"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+)
+
+// MskModuleBackendRule checks whether an MSK module has an S3 backend defined with a key that has as suffix the name of the team.
+type MskModuleBackendRule struct {
+	tflint.DefaultRule
+}
+
+// NewMskModuleBackendRule returns a new rule.
+func NewMskModuleBackendRule() *MskModuleBackendRule {
+	return &MskModuleBackendRule{}
+}
+
+// Name returns the rule name.
+func (r *MskModuleBackendRule) Name() string {
+	return "msk_module_backend"
+}
+
+// Enabled returns whether the rule is enabled by default.
+func (r *MskModuleBackendRule) Enabled() bool {
+	return true
+}
+
+// Severity returns the rule severity.
+func (r *MskModuleBackendRule) Severity() tflint.Severity {
+	return tflint.ERROR
+}
+
+// Link returns the rule reference link.
+func (r *MskModuleBackendRule) Link() string {
+	return ""
+}
+
+// Check checks whether.
```

Comment:
We already stated what this rule is doing in the `MskModuleBackendRule` docs, so possibly don't need anything here

```suggestion
```